wallet-connect-test-bench/front-end/src/constants.ts

```diff
@@ -0,0 +1,101 @@
+import {
+    BrowserWalletConnector,
+    ephemeralConnectorType,
+    Network,
+    WalletConnectConnector,
+    CONCORDIUM_WALLET_CONNECT_PROJECT_ID,
+} from '@concordium/react-components';
+import { SignClientTypes } from '@walletconnect/types';
+import moment from 'moment';
+
+export const REFRESH_INTERVAL = moment.duration(10, 'seconds');
+
+// The TESTNET_GENESIS_BLOCK_HASH is used to check that the user has its browser wallet connected to testnet and not to mainnet.
+export const TESTNET_GENESIS_BLOCK_HASH = '4221332d34e1694168c2a0c0b3fd0f273809612cb13d000d5c2e00e85f50f796';
+
+export const SET_U8_PARAMETER_SCHEMA = 'Ag==';
+
+export const SET_U16_PARAMETER_SCHEMA = 'Aw==';
+
+export const SET_CONTRACT_ADDRESS_PARAMETER_SCHEMA = 'DA==';
+
+export const SET_ADDRESS_PARAMETER_SCHEMA = 'FQIAAAAHAAAAQWNjb3VudAEBAAAACwgAAABDb250cmFjdAEBAAAADA';
+
+export const SET_ACCOUNT_ADDRESS_PARAMETER_SCHEMA = 'Cw==';
+
+export const SET_HASH_PARAMETER_SCHEMA = 'HiAAAAA=';
+
+export const SET_PUBLIC_KEY_PARAMETER_SCHEMA = 'HiAAAAA=';
+
+export const SET_SIGNATURE_PARAMETER_SCHEMA = 'HkAAAAA=';
+
+export const SET_TIMESTAMP_PARAMETER_SCHEMA = 'DQ==';
+
+export const SET_STRING_PARAMETER_SCHEMA = 'FgI=';
+
+export const SET_OPTION_PARAMETER_SCHEMA = 'FQIAAAAEAAAATm9uZQIEAAAAU29tZQEBAAAAAg==';
+
+export const SET_OBJECT_PARAMETER_SCHEMA =
+    'FAAMAAAACAAAAHU4X3ZhbHVlAgkAAAB1MTZfdmFsdWUDDQAAAGFkZHJlc3NfYXJyYXkQAhUCAAAABwAAAEFjY291bnQBAQAAAAsIAAAAQ29udHJhY3QBAQAAAAwNAAAAYWRkcmVzc192YWx1ZRUCAAAABwAAAEFjY291bnQBAQAAAAsIAAAAQ29udHJhY3QBAQAAAAwVAAAAYWNjb3VudF9hZGRyZXNzX3ZhbHVlCxYAAABjb250cmFjdF9hZGRyZXNzX3ZhbHVlDAoAAABoYXNoX3ZhbHVlHiAAAAAPAAAAc2lnbmF0dXJlX3ZhbHVlHkAAAAAQAAAAcHVibGljX2tleV92YWx1ZR4gAAAADwAAAHRpbWVzdGFtcF92YWx1ZQ0MAAAAb3B0aW9uX3ZhbHVlFQIAAAAEAAAATm9uZQIEAAAAU29tZQEBAAAAAgwAAABzdHJpbmdfdmFsdWUWAQ==';
```

Comment:
Can you just put a comment at the top explaining how you got these schemas. In acse we need to update them at some point.